tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "declaration": true,
     "declarationMap": true,
     "declarationDir": "types",
+    "esModuleInterop": true,
```

Comment:
Why was this added?

Reply:
i think I ran into some compability issue and this fixed it. Not sure if we actually need it though